assets/js/base/components/checkout/policies/style.scss

```
@@ -0,0 +1,22 @@
+.wc-block-components-checkout-policies {
+	text-align: center;
+	list-style: none outside;
+	line-height: 1;
+	margin: 2em 0 1em;
+}
+.wc-block-components-checkout-policies__item {
+	list-style: none outside;
+	border-right: 1px solid $gray-10;
+	display: inline-block;
+	padding: 0 0.25em;
+	margin: 0;
+
+	&:last-child {
+		border-right: 0;
+	}
```

If we want to avoid having to do a reset here, you could remove the `border-right` declaration in the parent selector and add it under `:not(:first-child)`:
```suggestion
.wc-block-components-checkout-policies__item {
	list-style: none outside;
	display: inline-block;
	padding: 0 0.25em;
	margin: 0;

	&:not(:first-child) {
		border-left: 1px solid $gray-10;
	}
```